docs/CODE_REVIEW.md

```diff
@@ -0,0 +1,130 @@
+# How to do code reviews for curl
+
+Anyone and everyone is encouraged and welcome to review code submissions in
+curl. This is a guide on what to check for and how to perform a successful
+code review.
+
+## All submissions should get reviewed
+
+All pull requests and patches submitted ot the project should be reviewed by
+at least one experienced curl maintainer before that code is accepted and
+received.
```

Comment:
"received" sounds a bit weird, how about "merged into the main development branch"?